### PR TITLE
alter markets so they are all in sync

### PR DESCRIPTION
**Pull request by edgethio**

### CODE REVIEW

File: src/stores/wagers.js [hunk 1]
```diff
@@ -24,6 +24,34 @@ const newWagerModel = {
...
 
+function matchMarkets(originalMarkets, expectedMarkets) {
+    return expectedMarkets.map(expectedMarket => originalMarkets.find(market => market.key === expectedMarket.key));
+}
+
+function normalizeMarkets(section) {
+    const eventMarkets = section.events
+        .map(e => e.displayGroups.markets);
+
+    if(eventMarkets.length < 1) return section;
```
Comment (donniemattingly) [inline src/stores/wagers.js:35]: this blows up if eventMarkets isn't defined either, maybe do a `!eventMarkets && <whatever>` here?

Comment (edgethio) [reply src/stores/wagers.js:35]: I think it's always defined.  You have sections without an event array at all?

**Comment by donniemattingly** [comment]
I deleted my other comment because it was mega-wrong - I do get `eventMarkets` as an `[undefined]` which causes some issues

**Comment by edgethio** [comment]
@donniemattingly can we run un-minified for these previews?

**Review by edgethio — COMMENTED** [review]
@donniemattingly 

> I deleted my other comment because it was mega-wrong - I do get `eventMarkets` as an `[undefined]` which causes some issues

forgot to reference index 0 of displayGroups :/

**Review by donniemattingly — APPROVED** [review]

**Pull request merged**